### DISCUSSION TITLE
Curso Figma gratis

### DISCUSSION THREAD

**Discussion by ChristianPineda** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#18] [Show and tell]
https://blog.facialix.com/cupon-udemy-megacurso-completo-de-figma-diseno-de-ui-ux-de-principiante-a-experto-con-100-de-descuento-por-tiempo-limitado/

Para quien quiera usar Figma para sus mockups 😊

**Comment by linettelopb** [comment]
Excelente muchas gracias por compartirlo. @ChristianPineda :)

**Comment by RicardoSolisS** [comment]
Llegué tarde, al usar el cupón dice
"Este cupón ha excedido el número máximo de canjes posibles y ya no puede utilizarse."
:-) igual no pasa nada, gracias y mucho éxito a todos.

**Reply by JosafatJimenezB** [reply]
ya no esta el de figma pero esta este de adobe XD por si les interesa https://www.youtube.com/watch?v=5HfqCSm9k3Y&list=PLhSj3UTs2_yVoP9-88WZI3HBS25vP6KoD&index=3

**Comment by OmarVenturaP** [comment]
Ya no alcance, pero igual, gracias por compartirlo. Saludos

**Reply by OmarVenturaP** [reply]
https://www.udemy.com/course/disena-una-increible-pagina-web-con-figma/

Encontré este por si les sirve... son 2 horas, espero y valgan la pena, jeje
